account_claims.go

```diff
@@ -209,14 +210,15 @@ func (a *AccountClaims) ClearRevocation(pubKey string) {
 	a.Revocations.ClearRevocation(pubKey)
 }
 
-// IsRevokedAt checks if the public key is in the revoked list with a timestamp later than
-// the one passed in. Generally this method is called with time.Now() but other time's can
-// be used for testing.
+// IsRevokedAt checks if the public key is in the revoked list with a timestamp later than the one passed in.
+// Generally this method is called with the subject and issue time of the jwt to be tested.
+// DO NOT pass time.Now(), it will not produce a stable/expected response.
 func (a *AccountClaims) IsRevokedAt(pubKey string, timestamp time.Time) bool {
 	return a.Revocations.IsRevoked(pubKey, timestamp)
 }
 
 // IsRevoked checks if the public key is in the revoked list with time.Now()
+// Deprecated: use IsRevokedAt instead.
```

Comment:
??? I would have thought that the server and other APIs would use `IsRevoked`, and the `IsRevokedAt` would be the underlying implementation, which can then be used by a test.

Reply:
IsRevoked uses the wrong time

Reply:
The issue is the API call, without the context of the account and the user JWT that API call should only be private API.

Reply:
so you are saying rename to isRevokedAt?

Reply:
To me, IsRevokedAt sounds like an internal API. - The public API should be IsRevoked()

Reply:
make it private

Reply:
For v1, we can't touch this one. Even "deprecated" is misleading since it will never be removed from v1, but can be in v2 since not yet released.
We could put a comment that explains to not use and that current behavior is wrong.